### PR TITLE
Automated AWS cloudtrail changes

### PR DESCRIPTION
**Pull request by esanyaCode**
**Details:**
This Change includes Creating a new cloudTrail in AWS, Upon successful creation it will Update, Stop and Delete the cloudTrail.

**Testing:**
Executed below tests in Ubuntu 20.04.

**Associated Issues:**
No issues observed in my testing. I am able to successfully create , Update, Stop and Delete cloudTrail in AWS.
CloudTrail_logs

### CODE REVIEW

**Comment by patel-bhavin** [comment]
Hey! this looks good after you updated, can you please add the policy document in a directory called `src` for example: `atomics/T1562.008/src/policy.json` and refer the same path in the yaml file. 

Also, move this command `aws s3 rb s3://#{s3_bucket_name} --force` under a new key called : cleanup_command 
Eg: https://github.com/redcanaryco/atomic-red-team/blob/6e4cbb9e85a5595cb3bb31a2fdfb14e3bd073886/atomics/T1003.002/T1003.002.yaml#L84

File: atomics/T1562.008/T1562.008.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      description: Name of the region
+      type: String
+      default: "us-east-1"
+  dependencies:
+    - description: |
+        Check if ~/.aws/credentials file has a default stanza is configured
+      prereq_command: |
+        cat ~/.aws/credentials | grep "default"
+        aws s3api create-bucket --bucket #{s3_bucket_name} --region #{region}
+        aws s3api put-bucket-policy --bucket redatomic-test --policy file://$PathToAtomicsFolder/T1562.008/src/policy.json
```
Comment (patel-bhavin) [inline atomics/T1562.008/T1562.008.yaml:29]: update redatomic-test with a variable!

Comment (esanyaCode) [reply atomics/T1562.008/T1562.008.yaml]: Updated as per review comment!

**Review by patel-bhavin — APPROVED** [review]

**Comment by patel-bhavin** [comment]
Awesome! i tested the changes and it works fine. Thanks for your contribution!

**Pull request merged**